vividus-engine/src/main/resources/org/vividus/engine/spring.xml

```diff
@@ -38,7 +38,9 @@
 
     <bean class="org.vividus.parser.StepExamplesTableParser" />
 
-    <bean id="testResourceLoader" class="org.vividus.resource.TestResourceLoader" />
+    <bean id="testResourceLoader" class="org.vividus.resource.TestResourceLoader">
+        <constructor-arg name="dynamicResourceSearchEnabled" value="${engine.dynamic-resource-search-enabled}" />
```

Comment:
it works only because there is only on boolean in the constructor, the args names are not preserved by default, so I would use index for safety